packages/addons/addon-depends/moonlight-common-c/package.mk

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="moonlight-common-c"
+PKG_VERSION="5b2cf1b8f7d94c729b4bbf702d4f9c4022381eb1"
+PKG_SHA256="6c9ed9b9dbbd8947fc3dc0801730a4ec9cc0d25364940682d6e4e5fa33a8bc92"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv3"
+PKG_SITE="https://github.com/moonlight-stream/moonlight-common-c"
+PKG_URL="https://github.com/moonlight-stream/moonlight-common-c/archive/${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION=""
+PKG_SHORTDESC="Shared C code for the implementation of Nvidia's GameStream protocol"
+PKG_LONGDESC="Shared C code for the implementation of Nvidia's GameStream protocol"
+PKG_TOOLCHAIN="manual"
```

Comment:
Please add the missing new line, thx
It's missing on multiple files

Reply:
Fixed, i think!